### PR TITLE
soma-home owl file is added to soma repository

### PR DESCRIPTION
**Pull request by AbhijitVyas**

### CODE REVIEW

**Comment by AbhijitVyas** [comment]
As discussed in last OTF meeting, ease_home is moved to SOMA repository as of now.

**Comment by daniel86** [comment]
> As discussed in last OTF meeting, ease_home is moved to SOMA repository as of now.

Please name the file "SOMA-HOME.owl"

File: owl/ease_home.owl [hunk 1]
```diff
@@ -0,0 +1,447 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns="http://www.ease-crc.org/ont/ease-home.owl#"
+     xml:base="http://www.ease-crc.org/ont/ease-home.owl"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+    <owl:Ontology rdf:about="http://www.ease-crc.org/ont/ease-home.owl"/>
```
Comment (daniel86) [inline owl/ease_home.owl:9]: - import of SOMA.owl is missing
- a small description would be nice (annotation property)
- the name should be rather "SOMA-HOME"

**Review by daniel86 — CHANGES_REQUESTED** [review]

File: owl/SOMA-HOME.owl [hunk 2]
```diff
@@ -0,0 +1,556 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns="http://www.ease-crc.org/ont/ease-home.owl#"
```
Comment (daniel86) [inline owl/SOMA-HOME.owl:2]: here is still ease-home, please change to SOMA-HOME.owl

File: owl/SOMA-HOME.owl [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+     xml:base="http://www.ease-crc.org/ont/ease-home.owl"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:SOMA="http://www.ease-crc.org/ont/SOMA.owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+    <owl:Ontology rdf:about="http://www.ease-crc.org/ont/SOMA-HOME.owl">
+        <owl:imports rdf:resource="http://www.ease-crc.org/ont/SOMA.owl"/>
+        <rdfs:comment>SOMA-HOME concentrates on the concepts that are specific to home, e.g. kitchen environment, household stuff.</rdfs:comment>
```
Comment (daniel86) [inline owl/SOMA-HOME.owl:12]: better: SOMA-HOME defines ...

Comment (daniel86) [inline owl/SOMA-HOME.owl:12]: better: household items

File: owl/SOMA-HOME.owl [hunk 4]
```diff
@@ -0,0 +1,556 @@
...
+        <rdfs:comment>A round, deep object used to store food or liquid.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#Bread -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#Bread">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#BakedGood"/>
+        <rdfs:comment>todo(DB): this seems too specific. Should it go to a more specific ontology? like &quot;SOMA-kitchen&quot;?</rdfs:comment>
```
Comment (daniel86) [inline owl/SOMA-HOME.owl:104]: this todo can be removed now

File: owl/SOMA-HOME.owl [hunk 5]
```diff
@@ -0,0 +1,556 @@
...
+        <rdfs:comment>A container used for cooking food.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#Pancake -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#Pancake">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#BakedGood"/>
+        <rdfs:comment>todo(DB): this seems too specific. Should it go to a more specific ontology? like &quot;SOMA-kitchen&quot;?</rdfs:comment>
```
Comment (daniel86) [inline owl/SOMA-HOME.owl:416]: the todo can be removed now

File: owl/SOMA-HOME.owl [hunk 6]
```diff
@@ -0,0 +1,556 @@
...
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#PancakeMix -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#PancakeMix">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#DesignedSubstance"/>
+        <rdfs:comment>todo(DB): this seems too specific. Should it go to a more specific ontology? like &quot;SOMA-kitchen&quot;?</rdfs:comment>
```
Comment (daniel86) [inline owl/SOMA-HOME.owl:425]: the todo can be removed

**Review by daniel86 — CHANGES_REQUESTED** [review]

**Comment by mpomarlan** [comment]
I have nothing to add to what Daniel said, so I didn't open a review.

File: owl/SOMA-HOME.owl [hunk 7]
```diff
@@ -0,0 +1,553 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns="http://www.ease-crc.org/ont/soma-home.owl#"
```
Comment (daniel86) [inline owl/SOMA-HOME.owl:2]: better use uppercase latter, not sure if it matters or not.
so use *SOMA-HOME.owl* exactly as I wrote it here.

**Review by daniel86 — CHANGES_REQUESTED** [review]

**Comment by daniel86** [comment]
@AbhijitVyas merge with master to get rid of error in OOPS, I think their servers are offline at the moment

**Review by daniel86 — APPROVED** [review]

**Pull request merged**